FloripaPublicTransportation/Route.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Route.swift
+//  FloripaPublicTransportation
+//
+//  Created by Marcelo Gobetti on 2/19/16.
+//  Copyright © 2016 Marcelo Gobetti. All rights reserved.
+//
+
+import Foundation
+
+class Route {
+    var id: Int?
```

Comment:
Another good point. When I was writing the very basic architecture, I hadn't included any constructors, so by default I had used optional members. Later I have included the inits and didn't pay attention that I could have removed the question marks.